flowmachine/flowmachine/core/connection.py

```diff
@@ -93,6 +94,10 @@ def __init__(
             pool_timeout=None,
             connect_args=connect_args,
         )
+        conn_id = md5(str(self.engine.url.host).encode())
+        conn_id.update(str(self.engine.url.port).encode())
+        conn_id.update(str(self.engine.url.database).encode())
+        self.conn_id = conn_id.hexdigest()
```

Comment:
Should probably add a comment for this - purpose it to give the conn a per-db id so we can disambiguate if you happen to use the same redis with multiple flowdbs.